C/Coin-OR/CHiPPS_BLIS/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+include("../coin-or-common.jl")
+
+sources = [
+    GitSource("https://github.com/coin-or/CHiPPS-BLIS.git", CHiPPS_BLIS_gitsha),
+]
+
+script = raw"""
+export CPPFLAGS="${CPPFLAGS} -I${includedir} -I${includedir}/coin"
+if [[ "${target}" == *mingw* ]]; then
+    export LDFLAGS="-L${bindir}"
+elif [[ "${target}" == *linux* ]]; then
+    export LDFLAGS="-ldl -lrt"
+fi
+cd $WORKSPACE/srcdir/CHiPPS-BLIS
+rm -f ${prefix}/lib/*.la
+update_configure_scripts
+sed -i s/elf64ppc/elf64lppc/ configure
+mkdir build
+cd build
+../configure \
+    --prefix=${prefix} \
+    --build=${MACHTYPE} \
+    --host=${target} \
+    --with-pic \
+    --disable-pkg-config \
+    --disable-debug \
+    --disable-dependency-tracking \
+    --enable-shared \
+    lt_cv_deplibs_check_method=pass_all \
+    --with-coindepend-lib="-lBcps -lAlps -lCgl -lOsiClp -lClp -lOsi -lCoinUtils"
+make -j${nproc}
+make install
+"""
+
+products = [
+    ExecutableProduct("blis", :blis),
+    LibraryProduct("libBlis", :libblis),
+]
+
+dependencies = [
+    Dependency("CoinUtils_jll", CoinUtils_version),
+    Dependency("Osi_jll", Osi_version),
+    Dependency("Clp_jll", Clp_version),
+    Dependency("Cgl_jll", Cgl_version),
+    Dependency("ALPS_jll", ALPS_version),
+    Dependency("BiCePS_jll", BiCePS_version),
+    Dependency("CompilerSupportLibraries_jll"),
+]
+
+build_tarballs(
+    ARGS,
+    "CHiPPS_BLIS",
```

Comment:
@giordano is it okay to name a JLL like this? 

There is a different https://github.com/JuliaBinaryWrappers/blis_jll.jl, and `CHiPPSBLIS` seemed a mouthful.

I guess the fact that jll's have `_` in their name means the Julia package manager is happy with it.

Reply:
Yes, it's ok for the package manager. I actually wondered whether we should have used CHiPPS in the name of the other packages, but I guess it's a bit too late now